autoload/handlers.vim

```diff
@@ -302,7 +302,21 @@ def s:processHoverReply(lspserver: dict<any>, req: dict<any>, reply: dict<any>):
     ErrMsg('Error: Unsupported hover contents (' .. reply.result.contents .. ')')
     return
   endif
-  hoverText->popup_atcursor({moved: 'word'})
+  if g:LSP_Hover_Reply_In_Preview
```

Comment:
Can you rename this setting to "g:LSP_Hover_In_Preview"?

Reply:
Done.